the-turing-way-community-online-book.md

```diff
@@ -0,0 +1,52 @@
+# The Turing Way: An open source community-driven online book about reproducible, ethical, inclusive and collaborative data science
+
+[_The Turing Way_](https://the-turing-way.netlify.app) started in December 2018 and has quickly evolved into a collaborative, inclusive and international endeavor with the aim of uncovering **gold standards to ensure reproducible, ethical, inclusive and collaborative data science**. How did this happen? I think two ingredients were central to _The Turing Way_'s success: **extraordinary community building and a clear enticing vision**.
+
+![image](https://zenodo.org/record/3695300/files/TooEasyNotToDo.jpg)
+
+
+## How do we develop The Turing Way?
+
+**Anyone can contribute** is a central theme. And not only that: anyone can bring ideas to the table. And folks are doing just that. At the time of writing this post 168 people have contributed.  So on average the project has gained **9 new [contributors](https://github.com/alan-turing-institute/the-turing-way#contributors) every month** since it's initiation.
+
+![image](https://zenodo.org/record/3695300/files/ThatCouldBeAChapter.jpg)
+
+**The extensive [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), the biweekly [collaboration cafes](https://hackmd.io/@KirstieJane/CollabCafe), and other events help people getting started. The [Code of Conduct](https://the-turing-way.netlify.app/community-handbook/coc.html) helps everyone feel save. All kinds of contributions are valued** (for details see the [all contributors specification](https://github.com/all-contributors/all-contributors)). I joined _The Turing Way_ at a book dash: a two day event where a room full of motivated contributors focus on improving the book and adding new content. It was a wonderful experience! [Kirstie Whitaker](https://www.turing.ac.uk/people/researchers/kirstie-whitaker) and [Malvika Sharan](https://about.me/malvikasharan) managed to create such a friendly and welcoming environment; It made working in that room in the Turing Institute with all the other contributors a blast.
```

Comment:
feel save -> feel safe

Reply:
Ugh I always make that mistake :sweat_smile: